{{cookiecutter.component_id}}/.github/workflows/jitpack_build.yml

```diff
@@ -0,0 +1,35 @@
+name: Trigger JitPack Build
```

Comment:
It would be nice to add it to https://github.com/hmcts/.github ? so everyone could reuse instead of copying it.

Reply:
Thanks @adusumillipraveen I'll do that